src/daemons/StorageDaemon.cpp

```diff
@@ -158,10 +158,18 @@ int main(int argc, char *argv[]) {
 
     auto ioThreadPool = std::make_shared<folly::IOThreadPoolExecutor>(FLAGS_num_io_threads);
 
+    std::string clusteridFile = paths[0] + "/storage.cluster.id";
+    auto clusterMan
+        = std::make_unique<nebula::meta::ClusterManager>("", clusteridFile);
+    if (!clusterMan->loadClusterId()) {
+        LOG(ERROR) << "storaged clusterId load error!";
```

Comment:
In loadClusterId() has print error log, and when sotraged has not been added, this print isn't error

Reply:
fixed！